examples/platform/silabs/SiWx917/BaseApplication.cpp

```diff
@@ -385,6 +371,8 @@ void BaseApplication::LightEventHandler()
 
 void BaseApplication::ButtonHandler(AppEvent * aEvent)
 {
+	uint8_t count = 5;
```

Comment:
Magic number here. Please change it to a define, or at least add some context to it..

Reply:
Yes I will add comment